syft/pkg/cataloger/apkdb/parse_apk_db_test.go

```diff
@@ -775,7 +775,7 @@ func TestMultiplePackages(t *testing.T) {
 				}
 			}()
 
-			pkgs, err := parseApkDB(file.Name(), file)
+			pkgs, _, err := parseApkDB(file.Name(), file)
```

Comment:
I went back and forth on this. There are a lot of places that ignore relationship data in test results now that [parser functions raise up artifact relationships](https://github.com/anchore/syft/blob/aba0de6e651d17a6e291e144cb92bdfd04693c69/syft/pkg/cataloger/common/parser.go#L11). I made certain to wire up anywhere that could raise up relationship data in the future in the cataloging execution path even if the cataloger in question does not surface relationships. However, I did not update the tests since this PR doesn't actually add any new relationships.

My take: any PR that is enhancing one or more catalogers to discover relationships should come with tests that prove the new functionality works. The lack of tests on a PR I think would be more of a signal than anything else.

Reply:
disregard... TODO statements here we come...

Reply:
👍  https://github.com/anchore/syft/pull/607/commits/628c2e44a5ce039a1a12b8f9748e8a3b1ca8d251